pontoon/tags/templates/tags/includes/tag.html

```diff
@@ -0,0 +1,37 @@
+{% import 'widgets/latest_activity.html' as LatestActivity %}
```

Comment:
As far as I can see this file is never used and so is the imported `tag_locales.html` file.